toxfile.py

```diff
@@ -33,3 +38,23 @@ def tox_on_install(
         return
     if BUILD_DIR.exists():
         shutil.rmtree(BUILD_DIR)
+
+
+@impl
+def tox_add_env_config(env_conf: EnvConfigSet, state: State) -> None:
+    env_conf.add_config(
+        keys=["globus_sdk_rmtree"],
+        of_type=list[str],
+        default=[],
+        desc="A dir tree to remove before running the environment commands",
+    )
+
+
+@impl
+def tox_before_run_commands(tox_env: ToxEnv) -> None:
+    sdk_rmtree = tox_env.conf.load("globus_sdk_rmtree")
+    for name in sdk_rmtree:
+        path = pathlib.Path(name)
+        if path.exists():
+            log.warning(f"globus_sdk_rmtree: {path}")
```

Comment:
Oh, quick review note: logged warnings show up in tox output by default, but info logs don't.